contents/select.md

```diff
@@ -0,0 +1,191 @@
+## Select {#sec:select}
+
+Whereas `filter` removes rows, `select` removes columns.
```

Comment:
does is actually removing? or creating a new data array?, not a view, right?

Reply:
I think that `filter!` and `select!` might be worth to mention

Reply:
> not a view, right?

Yes, you're correct. I'll try to word it a bit more careful.

> I think that filter! and select! might be worth to mention

I'll do it in the next round :+1: